ujt/dash_app.py

```diff
@@ -0,0 +1,22 @@
+""" Configuration for Dash app.
+
+Exposes app and cache to enable other files (namely callbacks) to register callbacks and update cache.
+App is actually started by ujt.py
+"""
+
+import dash
+import dash_bootstrap_components as dbc
+import dash_cytoscape as cyto
+from flask_caching import Cache
+
+# Initialize Dash app and Flask-Cache
+cyto.load_extra_layouts()
+app = dash.Dash(__name__, external_stylesheets=[dbc.themes.BOOTSTRAP])
+cache = Cache()
```

Comment:
This module is loaded before we start the server, and exposes app and cache for other modules to use.